R/helpers.R

```diff
@@ -47,22 +47,42 @@ lt_lake_sites <- function(which = c("all", "active", "inactive")) {
       "E206955", "0500124", "0500123",
       "E228889", "0603071", "0603006",
       "0603097", "0603017", "0603100",
-      "0803038", "0603019", "E262699"
+      "0803038", "0603019", "E262699",
+      "E275784", "1132490"
     ),
     inactive = c(
-      "E275784", "1131080", "1170009",
-      "1170011", "E207062", "1132490",
-      "E219178", "E213033", "E206793",
-      "E219459", "E216693", "E206789",
-      "E303412", "1199901", "1199902",
-      "1199904", "1199903", "E208723",
-      "E208721", "E208718", "E208722",
-      "E308528", "0603082", "0500129",
-      "E223053"
+      "1131080","1170009","1170011",
+      "E207062","E219178","E213033",
+      "E206793","E219459", "E216693",
+      "E206789","E303412", "1199901",
+      "1199902","1199904", "1199903",
+      "E208723","E208721", "E208718",
+      "E208722","E308528", "0603082",
+      "0500129","E223053"
     )
   )
 
   if (which == "all") return(unname(unlist(sites)))
 
   sites[[which]]
 }
+
+
+#' Long-term lake monitoring sites
+#'
+#' Get the REQ_IDs of all of the long-term lake monitoring sites.
+#'
+#' @param data loads a site ID lookup table
+#'
+#' @return a character vector of req ids (EMS IDs)
```

Comment:
It says EMS IDs here
```suggestion
#' @return a character vector of requisition ids 
```